### ISSUE TITLE
(LDA): change word-topic/doc-topic count type from Double to Int

### ISSUE THREAD

**Issue by hucheng** [cloudml/zen#35]
change word-topic/doc-topic count type from Double to Int.

**Comment by bhoppi** [comment]
This has been done already. And at the same time, the vector types used in code migrated from MLlib to Breeze

**Comment by hucheng** [comment]
Yeah, it would be worthwhile to use Breeze vector directly, rather than use MLlib vector indirectly.
